code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -450,22 +450,31 @@
 	reagent_state = LIQUID
 	color = "#C8A5DC"
 	metabolization_rate = 0.5 * REAGENTS_METABOLISM
-	overdose_threshold = 45
+	overdose_threshold = 30
 	addiction_threshold = 30
 
 /datum/reagent/medicine/ephedrine/on_mob_life(mob/living/M)
 	M.status_flags |= GOTTAGOFAST
-	M.AdjustParalysis(-1)
-	M.AdjustStunned(-1)
-	M.AdjustWeakened(-1)
-	M.adjustStaminaLoss(-1*REM)
+	M.AdjustParalysis(-0.4)
+	M.AdjustStunned(-0.4)
+	M.AdjustWeakened(-0.4)
+	M.adjustStaminaLoss(-0.5*REM)
 	..()
 	return
 
+/datum/reagent/medicine/ephedrine/on_mob_delete(mob/living/M)
+	M.visible_message("<span class='danger'>[M] suddenly runs out of breath!</span>")
+	M.adjustStaminaLoss(50*REM)
+
 /datum/reagent/medicine/ephedrine/overdose_process(mob/living/M)
 	if(prob(33))
 		M.adjustToxLoss(0.5*REM)
 		M.losebreath++
+	if(prob(12))
+		var/obj/item/I = M.get_active_hand()
+		if(I)
+			M.drop_item()
+			M.visible_message("<span class='danger'>[M] shudders and drops something on the floor!</span>")
```

Comment:
could just do "drops [I] on the floor!" instead of "something", your choice through.